i18n/web/fr.ts

```diff
@@ -166,6 +195,8 @@ const fr: WebTranslations = {
   govt: 'Gouvernement du Canada',
   yes: 'Oui',
   no: 'Non',
+  year: 'Anne',
```

Comment:
year: An
years: Ans

this is assuming masculine, a quick google suggests masculine is what we should use here: http://www.french-linguistics.co.uk/grammar/when_to_use_an_annee.shtml